packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncher.js

```diff
@@ -19,33 +22,71 @@ export const propTypes = {
   /** Callback called when application launcher toggle is clicked */
   onToggle: PropTypes.func,
   /** Indicates where menu will be alligned horizontally */
-  position: PropTypes.oneOf(Object.values(DropdownPosition)),
+  position: PropTypes.oneOf(Object.values(ApplicationLauncherPosition)),
   /** Adds accessible text to the button. Required for plain buttons */
   'aria-label': PropTypes.string
 };
 
+// seed for the aria-labelledby ID
+let currentId = 0;
```

Comment:
Is this incremented correctly when you have multiple of these on the same page? I think you'll want to use the generateId util instead

Reply:
It looks like it's incremented correctly. This is how Dropdown does it. (It's not new code.)